doc/index.rst

```diff
@@ -51,12 +51,31 @@ User manual
 Module reference
 -----------------
 
+.. currentmodule:: joblib.memory
+
+.. autosummary::
+   :toctree: generated/
+
+   Memory
+
+.. currentmodule:: joblib.parallel
+
+.. autosummary::
+   :toctree: generated/
+
+   Parallel
+
+.. currentmodule:: joblib.numpy_pickle
+
```

Comment:
This is the very hacky bit. I have no clue why this happens but just using the joblib.bla stuff doesn't create the link in the doc ...